lib/Adapter.js

```diff
@@ -21,6 +21,7 @@ var browser = require('bowser').browser,
 	canRenegotiate = false,
 	oldSpecRTCOfferOptions = false,
 	browserVersion = Number(browser.version) || 0,
+	Browser = { name: browser.name, version: browserVersion },
```

Comment:
This cannot be added as that info is already available via the `browser` object and `browserVersion` variable. Let me figure it out a simple refactor.